src/Container/Container.php

```diff
@@ -104,9 +104,10 @@ public function register($hint, $class, bool $singleton = false)
     {
         if ($class instanceof $hint[0]) {
             $this->hints[$this->parseHint($hint)] = ['class' => $class, 'singleton' => $singleton];
+        } else {
```

Comment:
If possible, please add the tests about container register failing on invalid/non-existing Class.

Reply:
@peter279k I added one more test for checking invalid class registration.